makefile

```diff
@@ -1,14 +1,29 @@
 DB_SERVICE = db  # Name of the PostgreSQL service in docker-compose
 DUMP_FILE = /app/phinvads.dump  # Name of your SQL dump file (within db container)
 
+.PHONY: start stop startlocal starttempl startair logs load refresh
+
 start:
-	@echo "Starting database container..."
+	@echo "Starting application..."
 	docker compose up -d
```

Comment:
should we add a `startdb` that just does the db?

Reply:
That's a good question. I had that thought at one point too and am certainly not against it. I think I was unsure if it'd be confusing to have a `startdb` but then use `stop` to shut it down (since a `stopdb` would do the same thing). What do you think? Does that seem fine?

Reply:
stop stops everything, including the db, so that doesn't seem that asymmetry doesn't personally bother me